ApiDocs/create.apidoc

```diff
@@ -21,7 +21,7 @@
   *         "q":"/configuration/createRepository",
   *         "sid": 1,
   *         "d":{
-  *         "token": "API-TOKEN",
+  *         "apiKey": "API-KEY",
```

Comment:
I assume in `apidoc` it should be `APIKey` instead of `apiKey`, since you are using `@JsonProperty("APIKey")`

Reply:
Ok, I'll change